content/posts/tau/index.md

```diff
@@ -107,7 +194,37 @@ Tau can be played on tablet or mouse with keyboard. Use the pointer to move a pa
 
 # settings
 
-You can see a full list of settings in [Tau option wiki page](https://github.com/Altenhh/tau/wiki/Options).
+Tau has a few options to customize your experience while playing this gamemode, you may have even stumbled upon this section within the option menus
+
+![Options](Images/Options.png)
+
+This page will serve as a kind of reference to what each customization option does, and to allow you to find the perfect setup.
+
+## Show Visualizer
+
+This option allows users to disable the visualizer that shows up around the ring during the gameplay at times.
+
+## Playfield dim
+
+This adjusts the background dim inside the playfield "Inside the ring" 
+
+*Basically putting it to a hundered makes it all black and putting it to 0 makes it go the same dim you have in osu! original settings*
+
+## Beat Size
+
+This adjusts the size of "Beats" notes to make them smaller in size or bigger based on the player's best interest
+
+*If you change the settings and you do not know how to restore it to the default just press the orange button that shows up near the setting*
+
+## Kiai Effect
+This options allows you to choose in between 3 different Kiai effects (the particles that show up in gameplay alongside the visualizer) and you will be able to choose in between the following three options:
+
+  1. Turbulent 
+    This is the default option, and it allows the particles to roam freely in the air of the screen and roam around it.
+  2. Classic 
+    This is the old classic version of the kiai effects, and it has a certain locked pattern where the particles are restrained.
+  3. Disabled
+    This option disables the particles entirely.
```

Comment:
![Screen Shot 2564-05-03 at 03 36 10](https://user-images.githubusercontent.com/68165621/116826919-b98c9200-abc0-11eb-9482-8de907f60677.png)
Change line to make header and description complete seperate.
```suggestion
  1. *Turbulent * : This is the default option, and it allows the particles to roam freely in the air of the screen and roam around it.
  2. *Classic * : This is the old classic version of the kiai effects, and it has a certain locked pattern where the particles are restrained.
  3. *Disabled* : This option disables the particles entirely.
```